### PR TITLE
Add onReady callback function to config

### LINKED ISSUE #77: Support for the event when the script is loaded.
Sometimes there is a need to perform an action after we know exactly when the script is loaded.

### PR DESCRIPTION
**Pull request by Shinigami92**
close #77

### CODE REVIEW

**Comment by Shinigami92** [comment]
@negezor Would you like to review this implementation?
Also please let me know if anything is missing or if something could be e.g. names differently?

I e.g. thought about renaming `onReady` to `onScriptLoaded` 🤔 

I also passed the `id` and the created `script`-element to the function. Is there anything else needed?

**Comment by negezor** [comment]
First of all, you need to call `onReady` after loading the script.
```ts
script.onload = () => {
  config.onReady?.(id, script);
};
```
You need to pay attention to the global installation of the script and call `onReady`. I also think it's better to pass options.
```ts
config.onReady?.({ id, script });
```
The rest is already left to implement in hooks for Vue.

**Comment by Shinigami92** [comment]
@negezor like so? I cannot test the `onload` call, cause jest seems not to call it :thinking: 
So could you please test it? Or should I release a beta? :thinking:

**Comment by Shinigami92** [comment]
@negezor I'm not sure if it is possible that way :thinking: 
I tried to create script elements via chrome browser console and used the `loaded` and `addEventListener('load', ...`. Both doesn't trigger :slightly_frowning_face:  
So I assume it's only possible when the HTML is first time rendered? But my package add them after HTML was initial loaded :thinking: I'm not sure...

**Comment by Shinigami92** [comment]
@negezor Nevermind... I just used an invalid `src` script and it needed to be valid to fire the event.

Still need to fake the event in jest, but it should work with the current implementation.
Also `addEventListener` is more "safe" cause `script.onload` could be reassigned.

File: src/utils.ts [hunk 1]
```diff
@@ -22,6 +36,13 @@ export interface LoadScriptOptions {
    * @see [Using Google Tag Manager with a Content Security Policy](https://developers.google.com/tag-manager/web/csp)
    */
   nonce?: string;
+  /**
+   * Will be called when the script is loaded.
+   *
+   * @param id GTM ID of the script.
```
Comment (negezor) [inline src/utils.ts:42]: All that remains is to update TSDoc 😅

**Review by negezor — APPROVED** [review]

File: src/utils.ts (hunk 1 above)
Comment (Shinigami92) [reply src/utils.ts:42]: Oh thx, good catch

**Pull request merged**